src/PKSim.Presentation/Presenters/Simulations/PopulationPKAnalysisPresenter.cs

```diff
@@ -34,33 +41,44 @@ public class PopulationPKAnalysisPresenter : PKAnalysisPresenter<IPopulationPKAn
       {
          _pkAnalysesTask = pkAnalysesTask;
          _exportTask = exportTask;
-         _allAnalyses = new List<PopulationPKAnalysis>();
+         _allAnalysesOnCurves = new List<PopulationPKAnalysis>();
+         _allAnalysesOnIndividuals = new List<PopulationPKAnalysis>();
          _populationPKAnalysisToDTOMapper = populationPKAnalysisToDTOMapper;
          _globalPKAnalysisPresenter = globalPKAnalysisPresenter;
          AddSubPresenters(_globalPKAnalysisPresenter);
          _view.AddGlobalPKAnalysisView(_globalPKAnalysisPresenter.View);
       }
 
-      public void CalculatePKAnalysis(IPopulationDataCollector populationDataCollector, ChartData<TimeProfileXValue, TimeProfileYValue> timeProfileChartData)
+      public void CalculatePKAnalysisOnCurves(IPopulationDataCollector populationDataCollector, ChartData<TimeProfileXValue, TimeProfileYValue> timeProfileChartData)
+      {
+         CalculatePKAnalysis(populationDataCollector, _allAnalysesOnCurves, _pkAnalysesTask.CalculateFor(populationDataCollector, timeProfileChartData), _allPKAnalysesOnCurves);
+      }
+
+      public void CalculatePKAnalysisOnIndividuals(IPopulationDataCollector populationDataCollector, IEnumerable<PopulationPKAnalysis> pks)
       {
-         _allAnalyses.Clear();
+         CalculatePKAnalysis(populationDataCollector, _allAnalysesOnIndividuals, pks, _allPKAnalysesOnIndividuals);
+      }
+
+      private void CalculatePKAnalysis(IPopulationDataCollector populationDataCollector, List<PopulationPKAnalysis> allAnalysis, IEnumerable<PopulationPKAnalysis> sourcePKs, List<PopulationPKAnalysis> targetPKs)
+      {
+         allAnalysis.Clear();
          _populationDataCollector = populationDataCollector;
-         _allPKAnalyses = _pkAnalysesTask.CalculateFor(populationDataCollector, timeProfileChartData);
-         _allAnalyses.AddRange(_allPKAnalyses);
+         targetPKs.Clear();
+         targetPKs.AddRange(sourcePKs);
+         allAnalysis.AddRange(targetPKs);
          LoadPreferredUnitsForPKAnalysis();
          BindToPKAnalysis();
-
          _globalPKAnalysisPresenter.CalculatePKAnalysis(new Simulation[] { populationDataCollector as Simulation });
       }
 
       protected override void BindToPKAnalysis()
       {
-         _view.BindTo(_populationPKAnalysisToDTOMapper.MapFrom(_allAnalyses));
+         _view.BindTo(_populationPKAnalysisToDTOMapper.MapFrom(_allAnalysesOnCurves), _populationPKAnalysisToDTOMapper.MapFrom(_allAnalysesOnIndividuals));
       }
 
       protected override IEnumerable<PKAnalysis> AllPKAnalyses
       {
-         get { return _allPKAnalyses.Select(x => x.PKAnalysis); }
+         get { return (View.IsOnCurvesSelected() ? _allPKAnalysesOnCurves : _allPKAnalysesOnIndividuals).Select(x => x.PKAnalysis); }
```

Comment:
I don't undertstand the IsOnCurvesSelected. What does this mean? Property boolean starting with Is should also be a property {get}

Reply:
> Property boolean starting with Is should also be a property {get}

Fixed

Reply:
> I don't undertstand the IsOnCurvesSelected.

Notice there is a view now with a tabbed view. One tab shows the calculation based on individuals and the other the same based on curves. When exporting to an excel, there were two options now: exporting both on different sheets or exporting only the selected tab. In the feature description we formalized it as "export only selected tab". So the presenter needs to know whether the selected tab is "OnCurves" or not.